bin/easystar-0.3.0.js

```diff
@@ -49,14 +49,15 @@ var EasyStar =
 	*   EasyStar.js
 	*   github.com/prettymuchbryce/EasyStarJS
 	*   Licensed under the MIT license.
-	* 
+	*
```

Comment:
You don't need to include this file. I will build it when I next bump the version.